tsconfig.json

```diff
@@ -1,28 +1,47 @@
 {
-	"compilerOptions": {
-		"target": "es5",
-		"lib": ["dom", "dom.iterable", "esnext"],
-		"allowJs": false,
-		"skipLibCheck": true,
-		"strict": false,
-		"forceConsistentCasingInFileNames": true,
-		"noEmit": true,
-		"esModuleInterop": true,
-		"module": "ESNext",
-		"moduleResolution": "node",
-		"resolveJsonModule": true,
-		"allowSyntheticDefaultImports": true,
-		"isolatedModules": true,
-		"jsx": "preserve"
-	},
-	"ts-node": {
-		// ts-node options here
-		"transpileOnly": true,
-		"compiler": "ttypescript",
-		// ts-node-specific overrides for compilerOptions
-		// any compilerOptions specified here will override the values specified above, *only* when running in ts-node
-		"compilerOptions": {},
-		"include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", "plop/ploppack.js"],
-		"exclude": ["node_modules"]
-	}
+  "compilerOptions": {
+    "target": "es5",
+    "lib": [
+      "dom",
+      "dom.iterable",
+      "esnext"
+    ],
+    "allowJs": false,
+    "skipLibCheck": true,
+    "strict": false,
+    "forceConsistentCasingInFileNames": true,
+    "noEmit": true,
+    "esModuleInterop": true,
+    "module": "ESNext",
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "allowSyntheticDefaultImports": true,
+    "isolatedModules": true,
+    "jsx": "preserve"
+  },
+  "ts-node": {
+    // ts-node options here
+    "transpileOnly": true,
+    "compiler": "ttypescript",
+    // ts-node-specific overrides for compilerOptions
+    // any compilerOptions specified here will override the values specified above, *only* when running in ts-node
+    "compilerOptions": {},
+    "include": [
+      "next-env.d.ts",
+      "**/*.ts",
+      "**/*.tsx",
+      "plop/ploppack.js"
+    ],
+    "exclude": [
+      "node_modules"
+    ]
+  },
+  "include": [
+    "next-env.d.ts",
+    "**/*.ts",
+    "**/*.tsx"
+  ],
+  "exclude": [
+    "node_modules"
+  ]
```

Comment:
I think I figured this out, I looked through our shared commit history and you had removed this line on Mar 28th in your "Merge pull request #8 from ethx-inc/antpowell/devops" after that Dahlia added the lines back on April 1st so I either had these lines in because they were in main from before you made your Merge on the 28th or because I pulled the lines again from when Dahlia re-added them on the 1st. either way I will remove them so that when this branch gets approved for merge, it will remove them from main!